### ISSUE TITLE
Cant char swap (event) gf after around Curstep 450

### ISSUE THREAD

**Issue by DaGhostyDude** [ShadowMario/FNF-PsychEngine#2010]
### Describe your bug here.

just making stuff, and swapping girlfriend out on the stage (I have around 2-3 gf swaps on the stage and they work fine). but after curstep 450 she just...vanishes when I try to swap her... I've been trying to fix it for days, (yes. i made it sure that I'm not writing the command wrong. multiple times.) I even edited the events Json manually, tried to only put a single gf change on the spot where it usually didn't work, but nothing works so I'm lead to believe that its a bug.

I hope I'm not the only one. tho if it isn't a bug and it happened to somebody before please help.

### Are you modding a build from source or with Lua?

Lua

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

nope.

**Comment by tarp2020** [comment]
I have the same problem as you lol ur not the only one with this problem

**Comment by NotEpik** [comment]
no, it's not about curStep or cutBeat or other stuff it's about how many times gf swap I swap like 2-3 times and then disappear too lol, real bug 😮

**Comment by DaGhostyDude** [comment]
Alright. I found a fix. 
It's not about how many swaps you have,
It's not about the curstep.
It seems like psych has some trouble when you swap to the same gf more in one chart. 

What I did is just create a duplicate characterjson. Literary opened the character and saved them without any change to mods -> characters just with another name. 

So at first they swap to the original and secondly to the clones
